packages/vscode-extension/src/chat/api.ts

```diff
@@ -0,0 +1,86 @@
+import * as vscode from "vscode";
+
+import { Logger } from "../Logger";
+import { getTelemetryReporter } from "../utilities/telemetry";
+
+const BASE_RADON_AI_URL = "https://radon-ai-backend.swmansion.com";
+
+export async function getSystemPrompt(userPrompt: string, jwt: string): Promise<any> {
+  try {
+    const url = new URL("/api/system_prompt/", BASE_RADON_AI_URL);
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "Authorization": `Bearer ${jwt}`,
+      },
+      body: JSON.stringify({ prompt: userPrompt }),
+    });
+
+    if (response.status !== 200) {
+      Logger.error(`Failed to fetch response from Radon AI with status: ${response.status}`);
+      getTelemetryReporter().sendTelemetryEvent("chat:error", {
+        error: `Failed to fetch with status: ${response.status}`,
+      });
+      return;
+    }
+    return await response.json();
+  } catch (error) {
+    if (error instanceof Error) {
+      Logger.error(error.message);
```

Comment:
would be better to print some additional context for the error log and not just the error message here. This way it may be easier to locate where the error occurred based on where it gets printed.

Reply:
Yessir 🫡 

Fixed in ac25c354b4b56dca74f4ed8eefd0f0da79d8566b